modules/status/sound_other.go

```diff
@@ -0,0 +1,60 @@
+// +build !openbsd,!freebsd,!netbsd
+
+package status
+
+import (
+	"log"
+
+	wmtheme "fyne.io/fynedesk/theme"
+	"github.com/mafik/pulseaudio"
+)
+
+// Destroy destroys the module
```

Comment:
Hmm perhaps some better wording here.  "Destroy destroys" just sounds odd.  Destroy removes the module maybe?

Reply:
This was copied from all other modules, I can review them all and update accordingly